config/crd/kustomization.yaml

```diff
@@ -59,6 +69,10 @@ patchesStrategicMerge:
 - patches/additional_app_label_in_crds.yaml
 # +kubebuilder:scaffold:crdkustomizeadditionalapplabelincrdspatch
 
+# [ActiveDoc CRD additional printer columns]. This patch adds spec.additionalPrinterColumns
+- patches/activedoc_additional_columns.yaml
```

Comment:
I think there's no need to provide a patch for printercolumns.

I've seen there's a kubebuilder CRD marker:
```
// + kubebuilder:printcolumn
```

that we might be able to use. See https://book.kubebuilder.io/reference/markers/crd.html for more details